client/store/data/section/slice.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, Reducer } from '@reduxjs/toolkit'
+import { fetchSectionData } from './actions/fetchSectionData'
+import { fetchSectionTablesMetadata } from './actions/fetchSectionTablesMetadata'
+import { SectionDataState } from './stateType'
+
+const initialState: SectionDataState = {
+  tables: [],
```

Comment:
tables are more metadata than data.
should we have tables under assessment state ? maybe it will be more consistent